README.md

```diff
@@ -51,6 +51,10 @@ require('iswap').setup{
   -- default 'Comment'
   hl_grey = 'LineNr',
 
+  -- Highlight priority for the greyed background
+  -- default '1000'
+  hl_grey = '200',
```

Comment:
I'm not sure if I'd like for this to be an option; it should override syntax highlighting IMO. Also, this should be `hl_grey_priority` right?

Reply:
> I'm not sure if I'd like for this to be an option; it should override syntax highlighting IMO

The user might have some other plugin which has a large priority, the option is given so that it overrides everything else